flowmachine/flowmachine/features/subscriber/active_subscribers.py

```diff
@@ -0,0 +1,120 @@
+import datetime
+from datetime import timedelta, date, datetime
+from typing import List, Union, Optional
+from flowmachine.core.query import Query
+from flowmachine.features.subscriber.call_days import CallDays
+from flowmachine.features.subscriber.interevent_interval import IntereventInterval
+from flowmachine.features.utilities.events_tables_union import EventsTablesUnion
+
+
+class ActiveSubscribers(Query):
+    """Returns a list of subscribers active `active_days`
+    out of `interval`, optionally with at least Z call-hours active"""
```

Comment:
Checking my understanding here:

A row of this table would be: the `date`, the `subscriber`, the number of days in `date-interval` - `date` they were active.

Active means 'observed `active_hours`/24 hours` for a given day?

Reply:
That's a precursor step; the row would be just `date` and `subscriber`, filtered where the number of days a subscriber is active in `date - interval` and `date` is >= `active_days`.
`active_hours` I've not implemented yet, but yes - `active_hours'/24 means a sub is active on that day.